soa/tracks/ml/1.md

```diff
@@ -1,15 +1,18 @@
 ## ML Track
-Welcome to the ML track. We hope you're really excited for this.
-For starters we'll brush up your Python Skills. This includes your understanding of
-- [Numpy](https://numpy.org/)
-- [Pandas](https://pandas.pydata.org/)
-- [Matplotlib](https://matplotlib.org/)
+Welcome to the ML track. We hope you're really excited for this.  
+For starters we'll brush up your Python Skills. This includes your understanding of  
 
-Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%201.ipynb) to view the Jupyter-Notebook.
-If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/).
+- [Numpy](https://numpy.org/)  
+- [Pandas](https://pandas.pydata.org/)  
+- [Matplotlib](https://matplotlib.org/)  
+
+Click [here](https://github.com/kabirnagpal/SoA-ML-14/blob/master/week%201.ipynb) to view the Jupyter-Notebook.  
+If you don't have any Python Environment, you can also try the code in [Google Colab](https://colab.research.google.com/).  
+
+
+How to get mean of each column in a Data Frame named `df`?  
```

Comment:
Shouldn't we tell people that this question is to be answered after they have gone through the notebook? Like it is now, the question is a little confusing.